bin/varnishtest/tests/o00005.vtc

```diff
@@ -0,0 +1,41 @@
+varnishtest "PROXY v2 TLV test"
+
+server s1 {
+	rxreq
+	txresp
+} -start
+
+varnish v1 -proto "PROXY" -vcl+backend {
+	import proxy from "${topbuild}/lib/libvmod_proxy/.libs/";
```

Comment:
from ... not needed any more

Reply:
in order to drop the from clause `bin/varnishtest/vmods.h` needs to be updated.

Reply:
fixed and rebase, thanks.